conf-sample.js

```diff
@@ -241,4 +241,5 @@ c.output  = {}
 // REST API
 c.output.api = {}
 c.output.api.on = true
+c.output.api.ip = // either a IPv4 or IPv6 address (as a string) to listen on, uses all available interfaces if omitted
```

Comment:
this should probably default to 0.0.0.0 or 127.0.0.1 as if the setting isn't set in the conf.js it falls back to conf-sample.js and currently I believe this would leave it undefined